### DISCUSSION TITLE
Question about connecting my laptop to BlueRetro

### DISCUSSION THREAD

**Discussion by Panja0** [darthcloud/BlueRetro#763] [Q&A]
I've just got the (recommended) Dreamcast BlueRetro by RetroScaler.
When I connect my laptop to the BlueRetro adapter (via blueretro.io) I always need to connect it 2 times.
When I press connect on the site (OTA, advanced config etc) I get the pairing screen, I choose the adapter and it will fail.
After that I press connect again and choose the adapter once again. After that it will connect.

First time connecting I have the adapter with a little signal bar/antenna in front.
Second time the adapter has a bluetooth logo.

Is this correct behaviour? 

Laptop is running Windows 11 and the Edge browser (Chromium based).
Also tried Google Chrome.

**Comment by darthcloud** [comment]
On some laptop / environment the web connection doesn't work well but on other it's flawless.

Try to to connect until it work, after 4-5 try close the tab and open new tab and try again.
Also try with another computer if you can.
